script/config.example.sh

```diff
@@ -106,3 +106,6 @@ COMPILE_ON_STARTUP=false
 
 # webhook URL for sending messages to Slack 'dev-sc-general' channel
 SLACK_WEBHOOK_SC_GENERAL=
+
+# Flag to enable/disable timelock controller
+USE_TIMELOCK_CONTROLLER=true
```

Comment:
I added this as a way to facilitate a transitionary period but let me know if we would rather just go all in and I can remove this